crates/router/src/connector/utils.rs

```diff
@@ -473,6 +474,12 @@ impl PaymentsPreProcessingData for types::PaymentsPreProcessingData {
     fn get_amount(&self) -> Result<i64, Error> {
         self.amount.ok_or_else(missing_field_err("amount"))
     }
+
+    // New minor amount function for amount framework
+    fn get_minor_amount(&self) -> Result<MinorUnit, Error> {
+        self.minor_amount.ok_or_else(missing_field_err("amount"))
+    }
```

Comment:
shouldn't it be 
```suggestion
        self.minor_amount.ok_or_else(missing_field_err("minor_amount"))
    }
```

Reply:
no we shouldn't send this as a part of response